test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -286,4 +286,19 @@ def test_successful_validate_pinless_debit_card
     assert_equal 'Y', response.params["status"]
     assert_equal true,  response.success?
   end
+
+  def test_network_tokenization_authorize_and_capture
```

Comment:
What is the experience like when somebody passed in a `NetworkTokenizationCreditCard` for a purchase? Looks like the tokenization parts just get ignored and its processed as though it were a credit card which, I believe, will result in a decline.

Now that we have a growing body of payment types, should we have a standard exception thrown to denote when a certain payment type is not valid for the given operation?